Tasks/DownloadBuildArtifactsV0/main.ts

```diff
@@ -389,6 +414,36 @@ function configureDownloaderOptions(): engine.ArtifactEngineOptions {
     return downloaderOptions;
 }
 
+function extractTars(downloadPath: string, tarArchivesPaths: string[]): void {
```

Comment:
Nit: could you please add some comments?

Reply:
Done